recipes/napari-blob-detection/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "napari-blob-detection" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-blob-detection-{{ version }}.tar.gz
+  sha256: a036c6f05e852ca0d332358249ec6c442c1ebdb83781e0609d9b146990e6f938
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - magicgui
+    - napari >=0.4.13
+    - numpy
+    - python >=3.8
+    - scikit-image
+
+test:
+  imports:
+    - napari_blob_detection
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/andy-sweet/napari-blob-detection
+  summary: Detects blobs in images
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - goanpeca
```

Comment:
Should this be updated to me, or you the actual maintainer on record here?

Reply:
@jaimergp and I will be listed as maintainers for napari plugins for a bit :)

Reply:
And now that you agreed I can add you :)